test/test.gypi

```diff
@@ -62,7 +62,6 @@
         'resources/mock_file_source.cpp',
         'resources/mock_file_source.hpp',
         'resources/mock_view.hpp',
-        'resources/resource_loader.cpp',
```

Comment:
Oh, I see. For some reason I was thinking that was the `ResourceLoader` source .cpp rather than a test file.